doc/source/getting_started/learning.rst

```diff
@@ -4,15 +4,39 @@
 Learning PyMAPDL
 ================
 
-Ansys has prepared multiple resources to help you to learn and use PyMAPDL.
-You can access them from the website `Ansys Learning Resources <learning_resources_>`_.
-In this page, you can find a summary of the available resources for MAPDL and PyMAPDL.
+Introduction
+============
+
+.. centered:: _PyMAPDL provides a Pythonic access to Ansys Finite Element Solver MAPDL._
```

Comment:
```suggestion
.. centered:: _PyMAPDL provides Pythonic access to Ansys Finite Element Solver MAPDL._
```
I've never seen this directive used before so will have to see what it looks like in the rendered doc. I am also unsure about the content. It makes it look like "Ansys Finite Element Solver MAPDL" is a product name. Maybe it should be:
.. centered:: _PyMAPDL provides Pythonic access to MAPDL, an Ansys finite element solver._